### ISSUE TITLE
Publish for MS Edge (Chromium)

### ISSUE THREAD

**Issue by TPS** [musically-ut/lovely-forks#67]
@musically-ut

Aka ChrEdge or Edgium, the current MS Edge can directly _use_ the Chrome extension & install from the Webstore, but not by default.  Closest available on Edge AddOn site is [Active Forks](https://microsoftedge.microsoft.com/addons/detail/active-forks/bpjmkkjkcligjfbanmjnlkjffalbehpb), but not so elegant.

Specific details on publishing to Edge AddOn site is @ https://docs.microsoft.com/microsoft-edge/extensions-chromium/publish/publish-extension

**Comment by musically-ut** [comment]
Interesting.

I personally do not use Windows; I'm downloading MS Edge now and will try to see if the port is as easy as they claim. 

If so, I'll release the same extension on Edge too; it is more or less stable and I don't think I'll be doing many major releases in the near future. So the overhead of supporting another platform should not be too much. 

Eventually, I hope GitHub itself will implement this feature or some variation thereof. :) 

Thanks for the suggestion!

**Comment by TPS** [comment]
I don't think you need to port _at all_, just republish.  As I mentioned, if 1 can toggle a setting, this installs, runs, & even updates directly from the Chrome store.  Publishing in the Edge store is for those unwilling, unknowing, or unable (e.g., setting might be unavailable due to enterprise policy) to do so.

**Comment by Dishavishwakarma** [comment]
Hi @musically-ut,

Microsoft Edge now provides developers an option to port their existing Chrome extension to Microsoft Edge Addons website with minimal code changes. You can find all the details about how to port your extension here - https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/port-chrome-extension.

If you have any more queries about porting Chrome extension to Microsoft Edge or anything else, you can contact the relevant teams whose links are given here - https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/contact-extensions-team.

Let me know if this was helpful. I am happy to help you with more details. 

Thank you!

**Comment by pm100** [comment]
@TPS I installed into edge from chrome store, but it doesnt work, could yo make it work?

**Comment by TPS** [comment]
@pm100 Sorry, I'm not the dev. @musically-ut?
